Resources/Prototypes/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -673,3 +673,30 @@
           Quantity: 4
         - ReagentId: Vitamin
           Quantity: 4
+
+- type: entity
```

Comment:
this should be in our file structure. so you want to make Resources/Prototypes/_Impstation/Entities/Objects/Consumable/Food/meals.yml and paste this entity there